shared/nav.android.js

```diff
@@ -93,13 +93,13 @@ class Nav extends Component {
   render () {
     const activeTab = this.props.tabbedRouter.get('activeTab')
 
-    if (this.props.config.navState === Constants.navStartingUp) {
-      return (
-        <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
-          <Text>Loading...</Text>
-        </View>
-      )
-    }
+    // if (this.props.config.navState === Constants.navStartingUp) {
```

Comment:
just so this doesn't complain. once this moves forward we should fix this up on mobile